bindings/nodejs/lib/Account.ts

```diff
@@ -536,6 +537,31 @@ export class Account {
         return JSON.parse(response).payload;
     }
 
+    /**
+     * Mint more native tokens.
+     * @param mintMoreNativeTokenOptions The options for minting more tokens.
+     * @param transactionOptions The options to define a `RemainderValueStrategy`
+     * or custom inputs.
+     * @returns The minting transaction and the token ID.
+     */
+    async mintMoreNativeToken(
+        mintMoreNativeTokenOptions: MintMoreNativeTokenOptions,
```

Comment:
Forgot to update the bindings

Reply:
No, https://github.com/iotaledger/wallet.rs/pull/1418#discussion_r966973844 :trollface:

Reply:
Oh sorry, don't think I received a notification for this one